src/bracket_checker/Node.java

```diff
@@ -0,0 +1,9 @@
+package bracket_checker;
+
```

Comment:
Why do you have this Node class?  This pattern keeps turning up in your Stack and Queue whiteboards.